### PR TITLE
Preparing data wrangling guide

### PR DESCRIPTION
**Pull request by gsilvapt**
This pull request will change the data wrangling guide structure to the one propose in issue #103 . 

The current dir called `/data` has files that represent the home page and other pages in the guide - not sure if it is worthy to create sub-dirs for these since they are just a few - and a larger directory called `/patterns`. Inside it, you can find the guides that were inside the same folder in a previous commit. 

Although Git says many files have been changed, most were just copied and pasted somewhere else. Only `/data/pattern/index.md` has been added. **No other file has been modified**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
     <div class="wrapper">
       <div>
         <h2>About Core Datasets Handbook</h2>
-        <p>These guides will introduce you to one of the principal projects of OKFN, currently community-driven. </p>
+        <p>These guides will introduce you to one of the principal projects of Open Knowledgee, currently community-driven. </p>
```
Comment (rufuspollock) [inline index.html:19]: Typo here ...

**Comment by rufuspollock** [comment]
@gsilvapt comments:
- small updates and improvements are not really part of  what this PR is doing (I guess you needed to submit because you did that stuff first - in that case i would have maybe just submitted a PR for that to get it in).
- Return to previous version seems to have a whole bunch of binary sass stuff in it. What exactly is that PR? I think you probably need to ignore .sass-cache and not be committing it but not absolutely sure.
- The final commit is what we actually want ...

So again I can't merge this. I wonder if you want to re-order your commits and/or start from scratch (re-fork this repo to a new location and copy over the changes you want, commit them, do PR and then repeat).

**Comment by gsilvapt** [comment]
@rgrp Confirm then this is what you wanted:
- I dropped the three commits above (this would reset things to a time frame before our discussion last weekend.
- Next, to fix issue #103, I would move old patterns to `/data/pattern/`.
- Commit, push and pull request these changes.

Would that be correct?

**Comment by rufuspollock** [comment]
@gsilvapt i think it would be really good to talk through so we are really clear on same page. My point here is first 2 commits in this PR had nothing to do with #103.

**Comment by gsilvapt** [comment]
@rgrp The requested to-do list is in the above comment. I will do that if you agree that is our way forward.

**Comment by rufuspollock** [comment]
@gsilvapt what above comment? My comment or your comment ...

I've suggested the way forward in terms of PRs here ...

**Comment by gsilvapt** [comment]
@rgrp  Mine, it is the only one with the to-do list.

We're trying to micromanage things that are causing a huge confusion to both. We should meet, discuss the project and move on from there - in my opinion.

**Pull request closed without merging**